protocol/message.go

```diff
@@ -46,5 +46,16 @@ type KeyLookupRequest struct {
 type KeyLookupProof struct {
 	Type int
 	AP   *merkletree.AuthenticationPath
-	TB   *merkletree.TemporaryBinding
+	TB   *merkletree.TemporaryBinding `json:",omitempty"`
+}
+
+type ConsistencyCheckRequest struct {
```

Comment:
This is just like a `KeyLookupRequest` except with an `Epoch` added. Couldn't we reuse `KeyLookupRequest` for the consistency checks and annotate `Epoch` with `omitempty`?